ccmlib/scylla_repository.py

```diff
@@ -303,6 +312,26 @@ def setup(version, verbose=True, skip_downloads=False):
     if skip_downloads:
         return directory_name(version), packages
 
+    if validate_by_hash:
+        if packages.scylla_unified_package:
+            new_hash = get_url_hash(packages.scylla_unified_package)
+        else:
+            # Old releases, there are no Scylla unified_package
+            # TODO: what's in this case?
+            new_hash = ""
```

Comment:
@fruch is it real case?

Reply:
add a check to packages.scylla_unified_package, and skip the hash checking if it doesn't exist

Reply:
added support of non-unified packages